doc/release-notes.md

```diff
@@ -0,0 +1,97 @@
+# Changes to `ga4gh/schemas` `master` branch since version `v0.5.1` (Oct 2, 2014)
+
+### Pervasive changes
+
+Renamed all protocols and records to remove the "`GA`" prefix from
```

Comment:
These are really Avro changes, since they don't actually affect the wire format. I think it would be helpful to break the changes into two sections, one for the wire format, that client side developers can use to see what they have to change, and another for the Avro biased summary that you have here.